demo/screens/SellOrderExecuteScreen.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { connect } from "react-redux";
+import { removeSellOrder } from "../actions";
+import SellOrderExecute from "@presentational/SellOrderExecute";
+import PropTypes from "prop-types";
+import ContractsABIs from "@constants/ContractsABIs";
+const { manaABI, estateABI, marketplaceABI } = ContractsABIs;
+import ContractsAddresses from "@constants/ContractsAddresses";
+const { manaAddress, estateAddress, marketplaceAddress } = ContractsAddresses;
+import { Action } from "tasit-sdk";
+const { Contract } = Action;
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+
+export class SellOrderExecuteScreen extends React.Component {
+  // TODO: Switch to new DecentralandEstate() once the SDK includes that
+  estateContract = new Contract(estateAddress, estateABI);
+  marketplaceContract = new Contract(marketplaceAddress, marketplaceABI);
+  manaContract = new Contract(manaAddress, manaABI);
+
+  _onOrderExecution = sellOrder => {
+    const { navigation } = this.props;
+    navigation.navigate("ListSellOrdersScreen");
+    this._executeOrder(sellOrder);
+  };
+
+  _executeOrder = async sellOrder => {
+    const { account, removeSellOrder } = this.props;
```

Comment:
I think I'd like to get the props outside of this function so that the function itself is purely functional and has these explicitly as parameters. Thoughts?

Reply:
Yes, make sense.

Reply:
Done.

Reply:
Thanks! Resolving